app/models/user.js

```diff
@@ -1,15 +1,19 @@
+'use strict';
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define(
-    'users',
+    'User',
     {
       firstName: { type: DataTypes.STRING, allowNull: false, field: 'first_name' },
       lastName: { type: DataTypes.STRING, allowNull: false, field: 'last_name' },
       email: { type: DataTypes.STRING, allowNull: false, field: 'email', unique: true },
       password: { type: DataTypes.STRING, allowNull: false }
     },
     {
-      timestamps: false
+      timestamps: true,
+      underscored: true
     }
   );
+  User.createUser = user => User.create({ ...user });
```

Comment:
We usually do not save this methods here but in a service. It is not bad to do it like this, but for convention we use to have a "logicless" model.

Reply:
Understood